<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="local" default="true"/>
    <arg name="pkg_name" value="timer"/>
    <arg name="node_name" default="timer_node"/>

    <group ns="$(arg veh)">   
    <!--remap from="timer_node/time_for_stop" to="stop_line_filter_node/time_for_stop"/-->
   
    <node if="$(arg local)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true">
        </node>

	<include unless="$(arg local)" file="$(find duckietown)/machines"/>     
	<node unless="$(arg local)" machine="$(arg veh)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true">
        </node>
    </group>
</launch>
